CMake/lrs_options.cmake

```diff
@@ -30,4 +30,5 @@ option(FORCE_RSUSB_BACKEND "Use RS USB backend, mandatory for Win7/MacOS/Android
 option(BUILD_NETWORK_DEVICE "Build Network Device support" OFF)
 option(FORCE_LIBUVC "Explicitly turn-on libuvc backend - deprecated, use FORCE_RSUSB_BACKEND instead" OFF)
 option(FORCE_WINUSB_UVC "Explicitly turn-on winusb_uvc (for win7) backend - deprecated, use FORCE_RSUSB_BACKEND instead" OFF)
-option(ANDROID_USB_HOST_UVC "Build UVC backend for Android - deprecated, use FORCE_RSUSB_BACKEND instead" OFF)
\ No newline at end of file
+option(ANDROID_USB_HOST_UVC "Build UVC backend for Android - deprecated, use FORCE_RSUSB_BACKEND instead" OFF)
+option(ENABLE_DEPTH_INVALIDATION "Turn on the depth frame validator" ON)
```

Comment:
Might want to mention it's for L500 only... maybe make it ENABLE_L500_DEPTH_INVALIDATION?
Also, it's not exactly clear when "depth invalidation" is -- maybe mention "to automatically catch corrupted frames and restart the sensor"?

Reply:
agreed.